### ISSUE TITLE
False positive `closure_spacing` violation

### ISSUE THREAD

**Issue by Kolos65** [realm/SwiftLint#4582]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Block comments in closure bodies trigger a closure_spacing violation starting from 0.50.0.
```swift
var tapped: (UITapGestureRecognizer) -> Void = { _ in /* no-op */ }
```

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint

error: Closure Spacing Violation: Closure expressions should have a single space inside each brace. (closure_spacing)
```

### Environment

* SwiftLint version: 0.50.0
* Installation method used: Homebrew
* Configuration file:

```yml
opt_in_rules:
  - closure_spacing
closure_spacing:
  severity: error
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations): **no**
* Which Xcode version are you using (check `xcodebuild -version`): **14.1 (14B47b)**

**Comment by jpsim** [comment]
Thanks for the report. Should be fixed in https://github.com/realm/SwiftLint/pull/4584
